ecosystem/gstreamer_plugin/gst_mtl_common.c

```diff
@@ -391,3 +391,35 @@ gboolean gst_mtl_common_parse_dev_arguments(struct mtl_init_params* mtl_init_par
   gst_mtl_port_idx++;
   return ret;
 }
+
+mtl_handle gst_mtl_common_mtl_init(struct mtl_init_params* p, StDevArgs* devArgs,
```

Comment:
```suggestion
mtl_handle mtl_gst_mtl_common_mtl_init_mtl(struct mtl_init_params* p, StDevArgs* devArgs, // mtl
```

Reply:
good suggestion lets try to fit more mtl in mtl in the next mtl release ok ?